rsocket-transport-netty/src/main/java/io/rsocket/transport/netty/server/WebsocketRouteTransport.java

```diff
@@ -74,9 +85,132 @@ public WebsocketRouteTransport(
                               connection, ByteBufAllocator.DEFAULT, mtu, false);
                     }
                     return acceptor.apply(connection).then(out.neverComplete());
-                  });
+                  },
+                  null,
+                  Math.max(DEFAULT_FRAME_SIZE, mtu == 0 ? FRAME_MAX_SIZE : mtu));
             })
         .bind()
         .map(CloseableChannel::new);
   }
+
+  static final class UriPathTemplate {
+
+    private static final Pattern FULL_SPLAT_PATTERN = Pattern.compile("[\\*][\\*]");
+    private static final String FULL_SPLAT_REPLACEMENT = ".*";
+
+    private static final Pattern NAME_SPLAT_PATTERN = Pattern.compile("\\{([^/]+?)\\}[\\*][\\*]");
+    // JDK 6 doesn't support named capture groups
+    private static final String NAME_SPLAT_REPLACEMENT = "(?<%NAME%>.*)";
+    // private static final String  NAME_SPLAT_REPLACEMENT = "(.*)";
+
+    private static final Pattern NAME_PATTERN = Pattern.compile("\\{([^/]+?)\\}");
+    // JDK 6 doesn't support named capture groups
```

Comment:
We are JDK8, so can be cleaned up.  Generally anything on Spring Reactor is JDK8 IIUC

Reply:
Yes - the java version of RSocket only supports 1.8 are greater so we don't need to worry about anything in JDK 1.6

Reply:
good catchup, I did not notice that when copy past that :D